rpcs3/evdev_joystick_handler.cpp

```diff
@@ -168,6 +168,30 @@ void evdev_joystick_handler::Close()
     }
 }
 
+inline int Deadzone(f32 input)
+{
+	int	deadzone=g_evdev_joystick_config.deadzone;
+	float deadzone_a=128.0/(128.0-float(deadzone));
+	float deadzone_b=255.0*(1.0-deadzone_a);
+	if (input>=128-deadzone&&input<=128+deadzone)
+	{
+		input=128;
+	}
+	else
+	{
+		if (input>128+deadzone)
+		{
+			input=deadzone_a*(float)input+deadzone_b;
```

Comment:
`deadzone_a` and b are only used here, they don't need to be declared at the start of the function.
Also, perhaps some more descriptive naming?

Reply:
Better naming would be great. I've been staring at this for a while now and I have no clue what either of these are doing.

Reply:
if you don't recalculate the output, the value will suddenly jump from 128 to 128+deadzone. I linearized the new range with y=ax+b. Changed the names in the new commit.